### ISSUE TITLE
some issues with exp_basic

### ISSUE THREAD

**Issue by zzm-tech** [thuml/Time-Series-Library#573]
thank you for spport for the repo

I found some unexcepted issue with code in exp_long_term_forecasting and debug in lines, which "output" is None in.

But every input have values. It confused long time, I hope some advice or information.

                if self.args.use_amp:
                    with torch.cuda.amp.autocast():
                        outputs = self.model(batch_x, batch_x_mark, dec_inp, batch_y_mark)

                        f_dim = -1 if self.args.features == 'MS' else 0
                        outputs = outputs[:, -self.args.pred_len:, f_dim:]
                        batch_y = batch_y[:, -self.args.pred_len:, f_dim:].to(self.device)
                        loss = criterion(outputs, batch_y)
                        train_loss.append(loss.item())
                else:
                    outputs = self.model(batch_x, batch_x_mark, dec_inp, batch_y_mark)

![Quicker_20241022_033828](https://github.com/user-attachments/assets/0ef60278-06d9-4d32-b98e-81401ea5ba51)
